README.md

```diff
@@ -51,6 +65,20 @@ export default YourPage
 | style | object | `{}` | Directly set the style object of the svg container |
 | lineStyle | object | `{}` | Set the style object of the line |
 
+### Donut
+
+| Property | Type | Default | Description |
+| - | - | - | - |
+| data | array | `[]` | An array of numbers |
+| width | string | - | CSS width of your chart |
+| height | string | `100%` | CSS height of your chart |
+| colors | string | `CATEGORICAL` | CSS colours used for each data point |
+| style | object | `{}` | Directly set the style object of the svg container |
+| offset | number | 0 | Starting point on circle for the first segment as a portion of a full rotation |
```

Comment:
This description is rather confusing, maybe just simplify to "Set the rotation offset between 0 and 1"